src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -102,18 +102,29 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 			let providerName = this.getProviderNameFromElement(element);
 			if (providerName === 'KUSTO') {
 				finalString = element.nodeTypeId !== 'Function' && escapedName.indexOf(' ') > 0 ? `[@"${escapedName}"]` : escapedName;
-			} else {
+			} else if (providerName === 'MSSQL') {
 				finalString = escapedSchema ? `[${escapedSchema}].[${escapedName}]` : `[${escapedName}]`;
+			} else if (providerName === 'PGSQL') {
+				finalString = escapedSchema ? `"${escapedSchema}"."${escapedName}"` : `"${escapedName}"`;
```

Comment:
You shouldn't be using the escaped schema/name here - that's only for bracket-delimited names.